services/bots/src/github-webhook/github-webhook.const.ts

```diff
@@ -75,12 +75,14 @@ export const entityComponents = new Set([
   'date',
   'datetime',
   'device_tracker',
+  'diagnostics',
```

Comment:
Diagnostic is a platform, not an entity component?

Reply:
Oh true

Reply:
```suggestion
```